tracer/src/Datadog.Trace/Telemetry/DTOs/ErrorData.cs

```diff
@@ -0,0 +1,26 @@
+﻿// <copyright file="ErrorData.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+namespace Datadog.Trace.Telemetry;
+
+internal readonly struct ErrorData
+{
+    public ErrorData(TelemetryErrorCode error)
+        : this(error, error.ToStringFast())
+    {
+    }
+
+    public ErrorData(TelemetryErrorCode code, string message)
+    {
+        Code = (int)code;
+        Message = message;
+    }
+
+    public int Code { get; }
+
+    public string Message { get; }
```

Comment:
Having a string code only would be better I believe. It would force us to standardize and send less data on the wire (like this kind of code: DDAS-0003-01)

Reply:
I think that's probably a non-starter, this has been the format for errors throughout v1 and v2, and would be breaking for all libraries.... 😬 

> On the contrary, after you create a referential table where you need it, and it's super easy to join both.

Which we have to create per library, and keep updated as new values are added. 😛 

This data is sent _once_ a minute. I just don't think it's worth the hassle personally 😉

Reply:
> Which we have to create per library, and keep updated as new values are added.

Nooo, that would be the start of sharing codes throughout libraries as ASM is doing :) You still don't know that I'm an utopian guy?